feature-wordflow/src/main/java/ru/coderedwolf/wordlearn/wordflow/di/WordFlowComponent.kt

```diff
@@ -59,24 +63,34 @@ interface WordFlowComponent : Injector<WordFlowFragment>,
 
 interface WordFlowDependencies : FlowDependencies {
     fun resourceProvider(): ResourceProvider
-    fun wordRepository(): WordRepository
+    fun wordDao(): WordDao
+    fun wordMapper(): WordMapper
     fun wordsCategoryInteractor(): WordsCategoryInteractor
     fun errorHandler(): ErrorHandler
 }
 
+@Module
+interface CreateWordModule {
+    @Binds
+    @PerFlow
+    fun providePhraseTopicRepository(
+        phraseTopicRepositoryImpl: WordRepositoryImpl
+    ): WordRepository
+}
```

Comment:
should to rename provide method to 'provideWordRepository' and argument too

Reply:
done